src/test/java/com/amazonaws/encryptionsdk/kms/KmsMasterKeyTest.java

```diff
@@ -99,8 +67,6 @@ public void testEncryptAndDecrypt() {
         when(supplier.get()).thenReturn(client);
 
         MasterKey otherMasterKey = mock(MasterKey.class);
-        when(otherMasterKey.getProviderId()).thenReturn(OTHER_PROVIDER_ID);
-        when(otherMasterKey.getKeyId()).thenReturn("someOtherId");
```

Comment:
Why are we able to remove this?

Reply:
I think this is a side effect of switching to a new version of our testing tools. They fail if you have unnecessary mocks (`when`s). So if the code is not required, you have to remove it or add a flag preventing the error.

Reply:
Specifically, you get

```
org.mockito.exceptions.misusing.UnnecessaryStubbingException: 
Unnecessary stubbings detected.
Clean & maintainable test code requires zero unnecessary code.
Following stubbings are unnecessary (click to navigate to relevant line of code):
```

Reply:
I just double checked this specific example on master (removing these two lines) and the test still passes.